package.json

```diff
@@ -91,6 +95,11 @@
     "pretest": "npm run build",
     "test": "npm run lint && karma start",
     "release": "./scripts/release.sh",
-    "start": "grunt"
+    "serve": "./scripts/open.js && parallelshell 'npm run watch' 'http-server -p 5678'",
```

Comment:
not sure what i was doing wrong, but for some reason in my initial testing *watch* and *http-server* both needed their own shell.  perhaps because i was using a double ampersand?  either way, happy to be able to clean this up.